diff --git a/texworks.appdata.xml b/texworks.appdata.xml.flathub
index 1e974a1..0dffdc8 100755
--- a/res/texworks.appdata.xml
+++ b/res/texworks.appdata.xml.flathub
@@ -45,6 +45,11 @@
 			Among other things, TeXworks features an integrated PDF previewer, synchronization
 			between the TeX files and the PDF output, and an extensible scripting interface.
 		</p>
+		<p>
+			To use the tex distribution provided by flathub, install it via
+
+			flatpak install org.freedesktop.Sdk.Extension.texlive//22.08
+		</p>
 		<p xml:lang="af">
 			TeXworks is 'n omgewing vir die skep van TeX (LaTeX, ConTeXt, ens.) dokumente, met 'n
 			Unicode-gebaseerde, TeX-bewuste redigeerder, geïntegreerde PDF besigtiger, en 'n skoon,
