diff --git a/getDefaultBinPaths.sh b/getDefaultBinPaths.sh
index ef432b6..db5a8b1 100755
--- a/getDefaultBinPaths.sh
+++ b/getDefaultBinPaths.sh
@@ -74,19 +74,20 @@ case "${PLATFORM}" in
 	*)			OS=$(echo "${PLATFORM}" | sed 's/.*-//')
 esac
 
-appendPath "/usr/local/texlive/2022/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2021/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2020/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2019/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2018/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2017/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2016/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2015/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2014/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2013/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2012/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2011/bin/$ARCH-$OS"
-appendPath "/usr/local/texlive/2010/bin/$ARCH-$OS"
+appendPath "/app/texlive/bin"
+appendPath "/var/run/host/usr/local/texlive/2022/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2021/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2020/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2019/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2018/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2017/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2016/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2015/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2014/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2013/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2012/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2011/bin/$ARCH-$OS"
+appendPath "/var/run/host/usr/local/texlive/2010/bin/$ARCH-$OS"
 
 for TEXLIVEROOT in /usr/local/texlive/* /opt/texlive/*; do
 	# Check if this is really a folder (e.g., /opt/... might not exist)
@@ -103,7 +104,7 @@ done
 
 # (3) append default paths that we should always check
 
-BINPATHS="$BINPATHS/usr/local/bin:/usr/bin"
+BINPATHS="$BINPATHS/var/run/host/usr/local/bin:/var/run/host/usr/bin"
 
 # strip leading and trailing colons from the list
 BINPATHS=$(echo "${BINPATHS}" | sed 's/^://;s/:$//')
